upgrade_versions.py

```diff
@@ -188,6 +193,21 @@ def _bump_release(self, pkg, log=None, user_name=None, user_email=None):
             assert user_email is not None
             rpm_bump_spec(pkg.specfile, log, user_name, user_email)
 
+    def _update_prerelease_tag(self):
+        if not self._prerelease:
+            self._prerelease = "%{nil}"
```

Comment:
For when the version changes from '4.9.0-rc4' to '4.9.0'. Not replacing would leave the '.rc4' there.